exporter/clickhouseexporter/exporter_traces_test.go

```diff
@@ -45,24 +45,6 @@ func TestExporter_pushTracesData(t *testing.T) {
 
 		require.Equal(t, 3, items)
 	})
-	t.Run("test scope name and scope version merge into span attribute", func(t *testing.T) {
-		initClickhouseTestServer(t, func(query string, values []driver.Value) error {
-			if strings.HasPrefix(query, "INSERT") {
-				if value, ok := values[9].(map[string]string); ok {
-					v, found := value[conventions.AttributeOtelScopeName]
-					require.True(t, found)
-					require.Equal(t, "io.opentelemetry.contrib.clickhouse", v)
-					v, found = value[conventions.AttributeOtelScopeVersion]
-					require.True(t, found)
-					require.Equal(t, "1.0.0", v)
-				}
-			}
-			return nil
-		})
-
-		exporter := newTestTracesExporter(t, defaultEndpoint)
-		mustPushTracesData(t, exporter, simpleTraces(1))
-	})
```

Comment:
why is this test removed?

Reply:
this test aims to test the scope name and scope version merge into span attribute, we no longer merge but use two new columns to store scopeName and scopeVersion